README.md

```diff
@@ -9,6 +9,8 @@ COPY *.map /etc/mapserver/
 COPY *.sh *.env /docker-entrypoint.d/
 ```
 
+The entry point should be `/etc/mapserver/mapserver.map`.
```

Comment:
s/entry point/main mapfile/

In a docker context, entry point is miss leading.